### ISSUE TITLE
Sorting issue on Chord diagrams

### ISSUE THREAD

**Issue by herekeo** [d3blocks/d3blocks#40]
Hello, 

I am tryin to plot a chord diagram, but the results are the plots with alphabetically sorted nodes. 
I need to plot the nodes as the order in the main dataframe.

Is there a way to do that?
Thanks!

**Comment by qhga** [comment]
Had a similar requirement today and took a closer look at this codebase.
The line that is responsible for sorting the diagram nodes alphabetically is here: [chord.js#1736](https://github.com/d3blocks/d3blocks/blob/fe31ede47f1e4acb6f02ca0cd75a865bcc036ee0/d3blocks/chord/d3js/chord.js#L1736)

It would be possible to make sorting optional by removing the `.sort(ascending)` from `const names = Array.from(new Set(data.flatMap(d => [d.source, d.target]))).sort(ascending);` and replace it with some if / else logic and an additional parameter that maybe could even take a function that is used in `.sort(my_func)`.

In my case, I just replace the `names` array with a static one, because the resulting set would contain the source and target keys in succession where I wanted the sources with the most connections first and following that everything else. Since I just required a one-shot graph I did not write anything reusable, sorry.

Maybe that gives you a hint where to look (:
